grpc.gemspec

```diff
@@ -29,7 +29,7 @@ Gem::Specification.new do |s|
   s.require_paths = %w( src/ruby/lib src/ruby/bin src/ruby/pb )
   s.platform      = Gem::Platform::RUBY
 
-  s.add_dependency 'google-protobuf', '~> 3.1'
```

Comment:
rather than a `'~> 3.7.0'`, this should probably be a `'~> 3.7'

`'~> 3.7.0'` will allow use of 3.8, 3.9, etc. `~> 3.7.0` will pin to `< 3.8`

Reply:
cc @jbolinger @dazume FYI- the next release of `grpc-tools` is planned to package `protoc` v3.7.0, which generates ruby `_pb.rb`'s that use `add_file`, which is available starting from the `google-protobuf` >= 3.7.0 runtime...... so `googleapis-common-protos-types` will probably need to bump its `google-protobuf` dep.

Reply:
Done.

Reply:
correction: cc @dazuma FYI and thanks for the heads up @apolcyn